searches/views.py

```diff
@@ -42,7 +42,21 @@ def get(self, request, query, sem_name, year):
         #     course_match_objs = apps.get_app_config('searches').searcher.vectorized_search(request.subdomain, query, sem)[:4]
         # else:
         #     course_match_objs = baseline_search(request.subdomain, query, sem)[:4]
-        course_match_objs = baseline_search(request.subdomain, query, sem).distinct()[:4]
+
+        # sorts queries by course number through a comparator
+        def course_comparator(course1, course2):
+            #isolate course number from XX.XXX.XXX
+            c1=str(course1)[7:10]
+            c2=str(course2)[7:10]
+            if c1 < c2:
```

Comment:
isn't c1 and c2 still strings? I would convert them to ints and then compare them